Cargo.toml

```diff
@@ -39,10 +39,11 @@ apca = "0.24.0"
 anyhow = {version = "1.0", default-features = false, features = ["std"]}
 chrono = {version = "0.4", default-features = false}
 chrono-tz = {version = "0.6.1", default-features = false, features = ["std"]}
+clap = { version = "3.1.18", features = ["derive", "env"] }
```

Comment:
What's the reason for the `env` feature if I may ask? The crate seems to be compiling without it, right? And I don't think we had environment variable support in the past.

I think we should consider setting `default-features = false`. Or do we want to have the colored help text? I'd rather keep things minimal, to be honest, and I see it as potentially getting in the way. E.g., on my terminal the colors are already questionable (yellow on white background is not the best to read). Also, `clap` seems to use `termcolor` for colored output, but we already pull in a different dependency for that (`yansi`). So I think if we keep the coloring we should consider migrating everything over to using `termcolor`, though that can be done later.

Reply:
I had actually only included it because of the `#[clap(version = env!("VERSION"))]` in args.rs, though I can't say that I tried without it. After removing, things still seem to work fine, so removed.